core/husarnet_manager.cpp

```diff
@@ -504,6 +513,7 @@ void HusarnetManager::startNetworkingStack()
   peerContainer = new PeerContainer(this);
 
   auto tunTap = Port::startTunTap(this);
+  this->tunTap = static_cast<TunTap*>(tunTap); //TODO: cleanup
```

Comment:
As we never intend to release this TUN/TAP… cleanup may be dead code

Reply:
I would just remove the cleanup remark. We need to have access to a `TunTap`'s incoming packet queue from the Husarnet stack context (`TunTap::processQueuedPackets()` called from the `HusarnetManager::processSocketEvents()`).